dex/src/test/scala/com/wavesplatform/dex/caches/RateCacheSpecification.scala

```diff
@@ -10,12 +10,11 @@ import org.scalatest.wordspec.AnyWordSpecLike
 import org.scalatestplus.scalacheck.{ScalaCheckPropertyChecks => PropertyChecks}
 
 import scala.concurrent.ExecutionContext.Implicits.global
-import org.scalatest.concurrent.ScalaFutures._
 import org.scalatest.time.{Millis, Seconds, Span}
 
 class RateCacheSpecification extends AnyWordSpecLike with Matchers with WithDb with MatcherSpecBase with PropertyChecks with NoShrink {
 
-  implicit val patienceConfig: PatienceConfig =
+  implicit override def patienceConfig: PatienceConfig =
```

Comment:
i think val is better here.

Reply:
val is better, but when we extend ScalaFutures, we also extends this chain: ScalaFutures -> Futures -> PatienceConfiguration, and PatienceConfiguration has a method:
```
implicit def patienceConfig: PatienceConfig = defaultPatienceConfig
```
so, we need to override it, or there may be conflict between two implicits with the same type

Reply:
Probably, @alexander-branevskiy meant `implicit override val` ?

Reply:
Suggest to remove this, we will use the default configuration from the base class

Reply:
Done